### ISSUE TITLE
Question: Any plans to use Adonis-Persona?

### ISSUE THREAD

**Issue by Maxximus007** [iamraphson/adonisjs-hackathon-starter#7]
The sign-up/login you've supplied works fine, so perhaps no real need to change it, but was wondering if you consider adding the new Adonis-Persona into the starter?

**Comment by iamraphson** [comment]
I will look into it..can you provide more information?

**Comment by Maxximus007** [comment]
Sure! https://github.com/adonisjs/adonis-persona

**Comment by Maxximus007** [comment]
It does have one advantage, the accountStates.

**Comment by niallobrien** [comment]
This would be a great addition. :+1:

**Comment by iamraphson** [comment]
@niallobrien @Maxximus007 so we should replace the present login flow with `Adonis-Persona`?

**Comment by Maxximus007** [comment]
It is not a drop-in replacement, so it will require some effort to change to `Adonis-Persona`. But I believe it could give the starter pack a valuable upgrade since it will use the package offered by Adonis itself.

**Comment by niallobrien** [comment]
I agree, I think the more Adonis specific packages used, the better.

**Comment by niallobrien** [comment]
Any progress on this?
